### PR TITLE
process balance on mobile

### PR DESCRIPTION
**Pull request by ManukMamyan**
no balance if mobile device and codec = h264

### CODE REVIEW

File: src/videoSendingBalancer‎/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+export const findVideoSender = (senders?: RTCRtpSender[]): RTCRtpSender | undefined => {
```
Comment (Krivega) [inline src/videoSendingBalancer‎/utils/index.ts:1]: remove optional senders

File: src/videoSendingBalancer‎/index.ts [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
   onSetParameters?: TOnSetParameters
 ) => {
   let mainCam: EEventsMainCAM | undefined;
   let resolutionMainCam: string | undefined;
 
-  const balance = (onSetParameters?: TOnSetParameters) => {
+  const balance = async (onSetParameters?: TOnSetParameters) => {
     const { connection } = sipConnector;
 
-    if (!connection) {
```
Comment (Krivega) [inline src/videoSendingBalancer‎/index.ts:23]: returtn this check

File: src/videoSendingBalancer‎/utils/index.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+};
+
+const findInResultByType = (results: RTCStatsReport, type: string) => {
+  return statsReportToArray(results).find((value) => {
+    return value.type === type;
+  });
+};
+
+const getCodecFromPeerConnection = (
+  senders?: RTCRtpSender[]
```
Comment (Krivega) [inline src/videoSendingBalancer‎/utils/index.ts:24]: from peerConnection and don't use optional params for target

File: src/videoSendingBalancer‎/utils/index.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      const codec = findInResultByType(stats, 'codec');
+
+      return codec?.mimeType;
+    });
+  }
+
+  return Promise.resolve();
+};
+
+export const hasBalanceVideo = async (
```
Comment (Krivega) [inline src/videoSendingBalancer‎/utils/index.ts:43]: use getCodecFromPeerConnection in src/videoSendingBalancer‎/index.ts

**Review by Krivega — CHANGES_REQUESTED** [review]

File: src/videoSendingBalancer‎/index.ts (hunk 2 above)
Comment (ManukMamyan) [reply src/videoSendingBalancer‎/index.ts:23]: +

**Review by Krivega — APPROVED** [review]

**Pull request merged**